src/main/resources/plugin.yml

```
@@ -32,11 +32,11 @@ permissions:
       imageonmap.rename: true
       imageonmap.removesplattermap: true
       imageonmap.delete: true
-      imageonmap.deleteother: false
+      imageonmap.deleteother: true
```

La description de ce groupe de permissions étant

> Grants all the user permissions (excluding administrative ones)

…je laisserais cette permission à `false`.

```suggestion
      imageonmap.deleteother: false
```

---

Oui en effet petite coquille de ma part.